### DISCUSSION TITLE
[Feature Request]: More versions for all services

### DISCUSSION THREAD

**Discussion by Tklaversma** [beyondcode/herd-community#488] [Ideas]
Would be nice if we could chose different versions on each service, so we can match production servers.

Especially for these:
- MariaDB
- Redis

Maybe even add the possibility to add our own versions of services?

**Comment by Tklaversma** [comment]
@sschlein you got any update regarding this feature request?

**Reply by tomcoonen** [reply]
MariaDB 11.4 woud be welcome

**Comment by mkduew** [comment]
We purchased our Pro version today, and that's the first thing I noticed. The latest MariaDB versions are missing. Will this issue be fixed? I see it's been open for two years already! :-(

**Comment by asim-blueprintprep** [comment]
Would be nice to see MariaDB 11.4 added as an option, as this is Magento 2's required version
